ambry-server/src/main/java/com/github/ambry/server/AmbryServer.java

```diff
@@ -263,16 +255,44 @@ public void startup() throws InstantiationException {
          * having a separate CloudtoStorageManager for cloud offers the flexibility to override methods and add more
          * suited to cloud.
          */
-        networkClientFactory = new RecoveryNetworkClientFactory(properties, registry, clusterMap, storageManager,
-            accountService);
-        vcrClusterSpectator = null; // Server does not talk to vcr during recovery
-        _recoveryManager =
-            new RecoveryManager(replicationConfig, clusterMapConfig, storeConfig, storageManager,
-                storeKeyFactory, clusterMap, scheduler, nodeId, networkClientFactory, registry, notificationSystem,
-                storeKeyConverterFactory, serverConfig.serverMessageTransformer, null,
-                clusterParticipants.get(0));
-        _recoveryManager.start();
-      } else {
+        networkClientFactory =
```

Comment:
The comment above doesn't seem to be up to date. We don't have CloudToStorageManager no more.

Reply:
good pt !